Makefile

```diff
@@ -3,12 +3,12 @@ CMDS  := $(addprefix bin/, $(shell go list $(MOD_FLAGS) ./cmd/... | xargs -I{} b
 
 .PHONY: build test vendor clean
 
-all: clean test build
+all: clean install-go-bindata test build
 
 $(CMDS):
 	go build $(MOD_FLAGS) $(extra_flags) -o $@ ./cmd/$(shell basename $@)
 
-build: clean $(CMDS)
+build: clean generate-migration-bundle $(CMDS)
```

Comment:
I don't think we want this as part of the `build` command so that CI will catch when we don't correctly commit the generated migration bundle to git.